oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/spi/binary/FulltextBinaryTextExtractor.java

```diff
@@ -193,13 +196,14 @@ public Void call() throws Exception {
       // Capture and report any other full text extraction problems.
       // The special STOP exception is used for normal termination.
       if (!handler.isWriteLimitReached(t)) {
-        log.debug(
-            "[{}] Failed to extract text from a binary property: {}."
+        String format = "[{}] Failed to extract text from a binary property: {}."
                 + " This is a fairly common case, and nothing to"
                 + " worry about. The stack trace is included to"
-                + " help improve the text extraction feature.",
-            getIndexName(), path, t);
+                + " help improve the text extraction feature.";
+        String indexName = getIndexName();
+        log.warn(format, throwableErrorFound ? new Object[]{indexName, path} : new Object[]{indexName, path, t});
```

Comment:
Here I think we should use info, because "This is a fairly common case" in the message

Reply:
done